server.js

```diff
@@ -1,5 +1,56 @@
-const app = require('./app')
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const mongoose = require("mongoose");
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+require("dotenv").config();
+
+const app = express();
+
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan(formatsLogger));
+
+const contactsRouter = require("./api/contacts");
+
+app.use("/api/contacts", contactsRouter);
+
+app.use((_, res) => {
+  res.status(404).json({ message: "Not found" });
+});
+
+app.use((error, _, res, __) => {
+  res.status(500).json({ message: error.message });
+});
+
+const PORT = process.env.PORT || 3000;
+const uriDb = process.env.DB_HOST;
+
+// Funtion used to connection with database and running server
+const connect = async () => {
+  try {
+    await mongoose.connect(uriDb, { dbName: "db-contacts" });
+    await app.listen(PORT, () =>
```

Comment:
```suggestion
     app.listen(PORT, () =>
```
Tu nie trzeba, callback jest tym co uzywasz.

Reply:
Jasne